clustered/integration-test/src/test/java/org/ehcache/clustered/management/ManagementActivator.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2011-2018 Software AG, Darmstadt, Germany and/or Software AG USA Inc., Reston, VA, USA, and/or its subsidiaries and/or its affiliates and/or their licensors.
+ * Use, reproduction, transfer, publication or disclosure is prohibited except as specifically provided for in your License Agreement with Software AG.
+ */
+package org.ehcache.clustered.management;
+
+import org.ehcache.management.ManagementRegistryService;
+import org.ehcache.management.registry.DefaultManagementRegistryConfiguration;
+import org.ehcache.spi.service.ServiceCreationConfiguration;
+import org.ehcache.spi.service.ServiceCreationConfigurationProvider;
+
+/**
+ * Activates management automatically if no config has been previously set manually or through XML.
+ *
+ * @author Mathieu Carbou
+ */
+public class ManagementActivator implements ServiceCreationConfigurationProvider<ManagementRegistryService> {
```

Comment:
So why not a `DefaultManagementRegistryConfiguration.defaultConfiguration()` instead?

Reply:
default is just equivalent of `new DefaultManagementRegistryConfiguration()`.

Here, i am just creating a sample use case to show you how it would work ;-) but i need to check the tests because some are failing probably because of the values here.

Reply:
I was giving an example for my comment about not needing a provider.

Reply:
Ok. I failed. It is a Java 9 feature. So the provider is necessary.